spec/support/helpers/tasks_helper.rb

```diff
@@ -47,7 +47,7 @@ def tasks_table_header
     end
 
     def tasks_table_content
-      tasks_table.all('tr')[1..-1].map { |row| row.all('td').map(&:text) }
+      tasks_table.all('tr').to_a[1..-1].map { |row| row.all('td').map(&:text) }
```

Comment:
This was done because `tasks_table.all('tr')[1..-1]` now always returns empty array.
I guess it's connected to override if `[]` in capybara and getting values from cache.